docs/Events.md

```diff
@@ -1,22 +1,42 @@
-Mock Firebolt: Events: Misc Notes
-=================================
+# Mock Firebolt: Events: Misc Notes
 
-Example event request, event ack, and event
+This document covers the flow of event registration, acknowledgement, and actual event messages in the Mock Firebolt framework. It also explains the configuration used to manage event messages.
+
+## Flow of Events
 
 ### Event Request
 
+An event request is a message from a client to register for an event. Below is an example of an event request message:
+
 ```
 {"jsonrpc":"2.0","method":"lifecycle.onInactive","params":{"listen":true},"id":1}
 ```
 
 ### Event Acknowledgement
 
+Once the server has successfully registered the client for the specified event, it optionally sends back an acknowledgement message. Here is an example:
+
 ```
 {"jsonrpc":"2.0","result":{"listening":true, "event":"lifecycle.onInactive"},"id":1}
 ```
 
 ### Event
 
+When the registered event occurs, the server sends an event message to the client. Here is an example of an event message:
+
 ```
 {"jsonrpc":"2.0","result":{"state":"inactive","previous":"foreground"}, "id":1}
 ```
+
+## Configuration
```

Comment:
We should add an example flow here. The user will need to know what they can access via handlebars and how. Walking through a flow using the default config in .mf.config.json would do nicely.